css/style.css

```diff
@@ -13,7 +14,7 @@
 
 body {
   background: #191765;
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   color: #fff;
 }
```

Comment:
Great design @AbigiyaTY 🤝

- [ ] [Optional] The contents of your design appear not to be well centered. You may kindly center the contents of your design accordingly so as to replicate the design provided in the Figma template.
![centering](https://user-images.githubusercontent.com/63917725/180785772-53b0fb93-5690-4798-b8c7-c18b52423a48.PNG)